core/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -151,6 +152,18 @@ public T addStreamTracerFactory(ServerStreamTracer.Factory factory) {
   public abstract T useTransportSecurity(File certChain, File privateKey);
 
   /**
+   * Makes the server use TLS.
+   *
+   * @param certChain InputStream containing the full certificate chain
+   * @param privateKey InputStream containing the private key
+   *
+   * @return this
+   * @throws UnsupportedOperationException if the server does not support TLS.
+   * @since 1.7.0
+   */
+  public abstract T useTransportSecurity(InputStream certChain, InputStream privateKey);
```

Comment:
We can't add new abstract methods to public classes. We need to convert to `File` or throw.